.github/workflows/ci.yml

```diff
@@ -196,7 +196,7 @@ jobs:
       earthly_runner_address: ${{ secrets.earthly_runner_address }}
       earthly_runner_secret: ${{ secrets.earthly_runner_secret }}
   deploy:
-    uses: input-output-hk/catalyst-ci/.github/workflows/deploy.yml@master
+    uses: input-output-hk/catalyst-ci/.github/workflows/deploy.yml@fix/ci-cli-build
```

Comment:
Should it be here and below `master` instead of `fix/ci-cli-build`?

Reply:
Will change back to master before merge. This is for testing.